src/main/java/com/twilio/taskrouter/WorkflowRule.java

```diff
@@ -1,18 +1,16 @@
 package com.twilio.taskrouter;
 
-import com.fasterxml.jackson.annotation.JsonAutoDetect;
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.*;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import com.google.common.base.MoreObjects;
 
 import java.io.IOException;
 import java.util.List;
 
+import static com.fasterxml.jackson.annotation.JsonInclude.*;
+
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
-@JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
+@JsonInclude(Include.NON_NULL)
```

Comment:
It looks like this `JsonInclude` is imported staticly, is this a distinction between this `JsonInclude` and the other ones?